som_leads_polissa/giscedata_crm_lead.py

```diff
@@ -67,6 +67,37 @@ def onchange_tipus_tarifa_lead(self, cursor, uid, ids, tipus_tarifa_lead):
             res["value"]["llista_preu"] = False
         return res
 
+    def create_entity_polissa(self, cursor, uid, crml_id, context=None):
+        res = super(GiscedataCrmLead, self).create_entity_polissa(
+            cursor, uid, crml_id, context=context
+        )
+
+        # recuperem la polissa recent creada del lead
+        polissa_id = self.read(
+            cursor, uid, crml_id, ['polissa_id'],
+            context=context
+        )['polissa_id'][0]
+
+        polissa_o = self.pool.get("giscedata.polissa")
+        polissa = polissa_o.browse(cursor, uid, polissa_id, context=context)
+        values = {}
+        if polissa.mode_facturacio != 'atr':
+            values['mode_facturacio_generacio'] = polissa.mode_facturacio
+
+        fp_id = polissa_o.calculate_fiscal_position_from_cups(
```

Comment:
Un comentari mes a nivell global, si la posicio fiscal la guardem en  el contracte pero apliquem el RD depenent de la data de la factura, te cap sentit que ho guardem al contracte? no ho podriem calcular cada cop parametritzant la funcio de sota amb la data i fer una taula amb intervals de dades com la que tenim a som_webforms?

Reply:
I que pensant-ho be, que no tinguem aquesta parametritzacio de la data, també fa que no podem unificar les dues funcions. :-(